swapper/swapper.py

```diff
@@ -106,3 +108,101 @@ def copy_and_replace(fc):
                 arcpy.management.ChangePrivileges(renamed_fc_sgid10, user, 'GRANT', 'AS_IS')
         except:
             print(f'could not update privileges to {renamed_fc_sgid10}')
+
+
+def compare():
+    '''compares data sets between SGID and SGID10 and returns the tables that are different
+    '''
+    dbo_owner = os.path.join(os.getenv('SWAPPER_CONNECTION_FILE_PATH'), 'SGID10', 'SGID10_sde.sde')
+
+    if not os.path.exists(dbo_owner):
+        print(f'{dbo_owner} does not exist')
+
+        return []
+
+    tables_needing_update = []
+
+    internal_connection = pyodbc.connect('DRIVER={SQL Server};SERVER=internal.agrc.utah.gov;DATABASE=SGID;UID=internal;PWD=internal')
```

Comment:
this probably should probably be removed...

Reply:
Done.